app/views/steps/_form.html.erb

```diff
@@ -59,10 +59,55 @@
 </div>
 
 <div class="form-group">
-  <div>
-    <p>
-      <%= step.broken_links %>
-    </p>
+  <div class="broken-links">
+    <% if step.broken_links && step.broken_links.count > 0 %>
+      <div class="panel panel-warning">
+        <div class="panel-heading">
+          <span class="glyphicon glyphicon-warning-sign" aria-hidden="true"></span>
+          <strong><%= step.broken_links.count %></strong> broken link<%= "(s)" if step.broken_links.count > 1 %> found.
+        </div>
+        <div class="table-responsive">
+          <table class="table table-bordered table-hover">
+            <thead>
+              <tr class="warning">
+                <th>URI</th>
+                <th>Status</th>
+                <th>Last checked</th>
+                <th>Problem summary</th>
+                <th>Errors</th>
+                <th>Warnings</th>
+                <th>Suggested fix</th>
+              </tr>
+            <thead>
+            <tbody>
+              <% step.broken_links.each do |link| %>
+                <tr>
+                  <td><%= link['uri'] %></td>
+                  <td><%= link['status'] %></td>
+                  <td><%= link['checked'] %></td>
+                  <td><%= link['problem_summary'] %></td>
+                  <td>
+                    <ul>
+                    <% link['errors'].each do |error| %>
```

Comment:
I'm just curious - does this ever output more than one error? Presumably a broken link can only return one status code, whether that's 404 or 500 or whatever.